pkg/process/checks/process.go

```diff
@@ -373,6 +374,12 @@ func (p *ProcessCheck) Run(nextGroupID func() int32, options *RunOptions) (RunRe
 		return p.run(nextGroupID(), false)
 	}
 
+	// No chunking for manual checks - set max batch size to max value to ensure one chunk
+	if options.NoChunking {
+		p.maxBatchSize = math.MaxInt32
+		p.maxBatchBytes = math.MaxInt32
```

Comment:
Let's use `math.MaxInt` here which will handle the architecture's default int size.